crates/ironrdp-cliprdr-native/src/windows.rs

```diff
@@ -112,18 +114,53 @@ pub struct WinClipboard {
 
 impl WinClipboard {
     /// Creates new clipboard instance.
-    ///
-    /// # Safety
-    ///
-    /// `window` must be a valid window handle to safely initialize clipboard processing.
-    /// The handle should be alive during the whole lifetime of the constructed clipboard instance.
-    pub unsafe fn new(window: HWND, message_proxy: impl ClipboardMessageProxy + 'static) -> WinCliprdrResult<Self> {
+    pub fn new(message_proxy: impl ClipboardMessageProxy + 'static) -> WinCliprdrResult<Self> {
```

Comment:
This breaks downstream code in `devolutions` repos, can we keep the old API? Does  `winit` not provide HWND in `v0.30` now?

Reply:
and the main issue, as I explained in the commit message, is that the window may not be yet created when RDP connection is set up.

Reply:
But `WinClipboard` still holds non-send HWND and this is by-design, why HWND in non-async method signature could cause issues? 🤔 

I don't like the idea of implicit hidden window creation here in cliprdr crate, I suggest to keep the old `new` method signature for backward compatibility (and keep `Safety` comment) , and moving hidden windows creation logic to `client`

Reply:
It's also unsafe to not own the HWND. Hidden window for clipboard is used by a lot of users, including GTK, spice agent, rust clipboard-win, just to name a few I know.

Reply:
@awakecoding WDYT, is it worth to use implicit hidden window for clipboard as @elmarco suggests?

Reply:
@elmarco I discussed this with @awakecoding internally, and we agreed that the hidden windows approach is indeed superior! 👍